charts/fybrik/templates/manager-deployment.yaml

```diff
@@ -106,6 +106,8 @@ spec:
               value: {{ .Values.manager.tls.certs.certSecretName }}
             - name: CERT_SECRET_NAMESPACE
               value: {{ .Values.manager.tls.certs.certSecretNamespace }}
+            - name: RESOURCE_POLLING_INTERVAL
```

Comment:
Why here and not in the config map used by manager?

Reply:
OK I can change that. Thanks